test/e2e/e2e_test.go

```diff
@@ -37,9 +44,15 @@ func handleFlags() {
 	flag.Parse()
 }
 
+// must be called after flags are parsed
+func initDockerImage() {
+	dockerImage = fmt.Sprintf("%s:%s", *dockerRepo, *dockerTag)
+}
```

Comment:
Maybe we should just make `dockerImage` a function, would drop the kinda useless variable. WDYT?

```suggestion
// must be called after flags are parsed
func dockerImage() string {
	return fmt.Sprintf("%s:%s", *dockerRepo, *dockerTag)
}
```

Reply:
Sure. let me just fix all the reference in the code